src/daemon/rpc_command_executor.cpp

```diff
@@ -2314,7 +2330,7 @@ bool t_rpc_command_executor::print_sn_status()
   }
 
   std::string const &sn_key_str = res.service_node_pubkey;
-  bool result = print_sn({sn_key_str});
+  bool result = print_sn({sn_key_str}, false);
```

Comment:
This command should also accept +json as well since it is just a wrapper around print_sn.